src/test/java/io/strimzi/kafka/bridge/http/ProducerTest.java

```diff
@@ -216,6 +222,8 @@ void sendBinaryMessageWithKey(VertxTestContext context) {
         consumer.subscribe(topic, done -> {
             if (!done.succeeded()) {
                 context.failNow(done.cause());
+            } else {
+                context.completeNow();
```

Comment:
This is wrong. If subscription fails that the test has to fail otherwise the test complete when the consumer gets the messages which happens in the previous handler. With this addition the test pass just because after subscription you are calling completeNow. The consumer doesn't have time to get messages that the test is shutdown and the handler code is not executed.

Reply:
ok, removed wrong code, and disabling this test if using external bridge